common/src/main/java/fudge/notenoughcrashes/utils/CrashLogUpload.java

```diff
@@ -46,33 +49,109 @@ public GistFile(String content) {
         }
     }
 
-    public static String upload(String text) throws IOException {
-        // Don't give a fuck about your preferences
-        return uploadToGist(text);
-//        ModConfig.CrashLogUploadType type = ModConfig.instance().uploadCrashLogTo;
-//        if (type == ModConfig.CrashLogUploadType.GIST) {
-//            return uploadToGist(text);
-//        } else {
-//            return uploadToHaste(text);
-//        }
+    private static ArrayList<ModConfig.CrashLogUploadType> fallBackTypes;
```

Comment:
I'm going to change this a bit after pulling I think

Reply:
👍